middlewares/validators/validator.helper.js

```diff
@@ -198,8 +204,9 @@ function stringValidator(fieldLocation, fieldname, optional = true) {
     const name = setProperValidationChainBuilder(
         fieldLocation,
         fieldname,
-        "invalid string"
+        "Invalid String"
```

Comment:
Nit: Maybe change others as well to capitalize, or leave them all uncapitalized?